### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by BryanSLam**
We're using gofig now bitch

Fixed all the weird typing stuff my code editor was complaining about
Removed the environment variables, they were stupid

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"StockAPIURL": "https://api.iextrading.com/1.0/stock/",
```
Comment (jonwho) [inline config.json]: im fine with endpoints being hardcoded but the error messages are more fun configurable

https://12factor.net/config

File: main.go [hunk 2]
```diff
@@ -183,7 +191,7 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 			s.ChannelMessageSend(m.ChannelID, outputJSON)
 			defer resp.Body.Close()
 		} else {
-			s.ChannelMessageSend(m.ChannelID, err_invalid_cmd)
+			s.ChannelMessageSend(m.ChannelID, config.InvalidCommandMessage)
```
Comment (jonwho) [inline main.go]: keep the error messages as env vars so i can call you stupid, stupid

**Comment by BryanSLam** [comment]
I dont get why you'd wanna keep them an environment vars wheb we can keep
it as a config option so there's a default option

Adding an abritrary step in the setup where the person has to add an
environment variable when we could just get a default seems like bad
practice

**Comment by BryanSLam** [comment]
When we could set a default and let them configure it themselves*

**Pull request merged**